objects/_objects.crop.scss

```diff
@@ -126,7 +127,21 @@ $inuit-crops: (
       @error "`#{$consequent}` needs to be a number."
     }
 
-    .o-crop--#{$antecedent}\:#{$consequent} {
+    $antecedent-class: $antecedent;
+    @if str-index(inspect($antecedent), ".") {
```

Comment:
Nice job @csshugs. While this does the job, what I would suggest adding are some comments explaining what this chunk of code does, so that anyone looking under the hood would have easier time understanding what they are looking at.

Reply:
You're absolutely right.